### DISCUSSION TITLE
Backup and Restore docs

### DISCUSSION THREAD

**Discussion by data-dude** [Infisical/infisical#720] [General]
Is there documentation on how to do a backup and restore?

**Comment by alberk8** [comment]
How does one backup and restore infisical secrets?

**Comment by vincenthartmann** [comment]
Hello, did you find a solution ? My temporary solution is to backup persistent volume of redis and mongoDB but it's not very easy to quicky restore

**Reply by data-dude** [reply]
We ended up just using Kyverno to sync secrets:

https://release-1-8-0.kyverno.io/policies/other/sync_secrets/sync_secrets/

We already use Kyverno for other things
